src/kde/quick_index.cc

```diff
@@ -456,7 +455,7 @@ quick_index::window_coarse_index_list(const std::array<double, 3> &window_min,
   if (spherical && (window_min[1] < 0.0 || window_max[1] > 2.0 * rtt_units::PI)) {
     // Only one bound of the window should every overshoot zero
     Check(!(window_min[1] < 0.0 && window_max[1] > 2.0 * rtt_units::PI));
-    size_t overlap_nbins = 1;
+    // size_t overlap_nbins = 1;
```

Comment:
@clevelam Can you confirm that this variable is not used?

Reply:
> 
> 
> > Otherwise it looks good.
> 
> Should be fixed by [c156e7f](https://github.com/lanl/Draco/commit/c156e7f6931f70a0f164270c84af24f3a73f84a1).

Early exit got me.  Let me run the whole set of `clang-tidy` before submitting another update.